### PR TITLE
Fix configs

### PR DESCRIPTION
**Pull request by athal7**
cc @eheydrick

### CODE REVIEW

**Comment by majormoses** [comment]
just needs a CHANGELOG entry

**Comment by athal7** [comment]
@majormoses thanks! updated with changelog entry

File: bin/check-mysql-query-result-count.rb [hunk 1]
```diff
@@ -38,6 +38,12 @@ class MysqlQueryCountCheck < Sensu::Plugin::Check::CLI
          description: 'MySQL password',
          default: ''
 
+  option :database,
+         short: '-d DATABASE',
+         long: '--database DATABASE',
+         description: 'MySQL database',
+         default: ''
```
Comment (majormoses) [inline bin/check-mysql-query-result-count.rb]: maybe we should have a sane default like mysql?

Comment (majormoses) [inline bin/check-mysql-query-result-count.rb]: @athal7 either I think we should have a sane default or make it required with no default unless there is a use case that does not need a database to query against.

Comment (athal7) [inline bin/check-mysql-query-result-count.rb]: @majormoses agreed, updated

**Pull request merged**